.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -135,6 +135,13 @@ jobs:
               ${{ inputs.test-cluster-name }} \
               ${{ env.AWS_DEFAULT_REGION }} \
               ${{ env.SAMPLE_APP_NAMESPACE }}
+
+              # If the workflow provides a specific ADOT image to test, record the Image Names in Env to verify if any change after patch
+              if [ ${{ inputs.appsignals-adot-image-name }} != "" ]; then
```

Comment:
Nit: The If statement on line 140 and 147 are the same. Could we move the content of the statement inside line 147? We could run it before the kubectl patch command.

Reply:
I do believe split these two part will have a more clear construct of code